.github/workflows/WebAutomatedTests.yml

```diff
@@ -48,12 +48,13 @@ jobs:
             - name: "Fixing files permissions"
               if: failure()
               run: |
-                  sudo find ${{ github.workspace}}/packages/pluggableWidgets -type d -exec chmod 755 {} \;
-                  sudo find ${{ github.workspace}}/packages/pluggableWidgets -type f -exec chmod 644 {} \;
-            - name: "Archive test screenshot results"
+                  sudo find ${{ github.workspace}}/packages/{pluggableWidgets|theming} -type d -exec chmod 755 {} \;
+                  sudo find ${{ github.workspace}}/packages/{pluggableWidgets|theming} -type f -exec chmod 644 {} \;
+            - name: "Archive test screenshot diff results"
               uses: actions/upload-artifact@v2
               if: failure()
               with:
                   name: test-screenshot-results
-                  path: ${{ github.workspace}}/packages/pluggableWidgets/**/tests/e2e/screenshot-results/diff/*.png
+                  path: |
```

Comment:
Can you explain what does the "pipe | " do here ? I am curious

Reply:
My bad, removed!